src/screens/AddressDetail.js

```diff
@@ -263,13 +270,23 @@ class AddressDetail extends React.Component {
     });
   }
 
+  getSelectedTokenMetadata = (selectedToken) => {
+    metadataApi.getDag(selectedToken).then((data) => {
+      if (data) {
+        this.setState({ selectedTokenMetadata: data });
+      }
+    }).finally(() => {
+      this.setState({ metadataLoaded: true });
```

Comment:
What if an error occurs? Should we still set `metadataLoaded` to true?

Reply:
If the metadata API returns 404 it is loaded as well. But maybe I should capture the error and consider it loaded only if it's 404

Reply:
Done